playbook/app/pb_kits/playbook/pb_multi_level_select/_multi_level_select.tsx

```diff
@@ -313,68 +377,121 @@ const MultiLevelSelect = (props: MultiLevelSelectProps) => {
     )
   }
 
-
-
   return (
-    <div {...ariaProps} {...dataProps} className={classes} id={id}>
-      <div ref={dropdownRef} className='wrapper'>
-        <div className='input_wrapper' onClick={handleInputWrapperClick}>
-          <div className='input_inner_container'>
-            {returnedArray.length !== 0 && returnAllSelected
-              ? returnedArray.map((item) => (
-                  <input type='hidden' name={`${name}[]`} value={item.id} />
-                ))
-              : null}
-
-            {returnedArray.length !== 0 && inputDisplay === "pills" && returnAllSelected
-              ? returnedArray.map((item, index) => (
-                  <FormPill
-                    key={index}
-                    text={item.label}
-                    size='small'
-                    onClick={(event: any) => handlePillClose(event, item)}
+    <div
+        {...ariaProps}
+        {...dataProps}
+        className={classes}
+        id={id}
+    >
+      <div
+          className="wrapper"
+          ref={dropdownRef}
+      >
+        <div
+            className="input_wrapper"
+            onClick={handleInputWrapperClick}
+        >
+          <div className="input_inner_container">
+            {variant === "single" && defaultReturn.length !== 0
```

Comment:
This logic is for the hidden input in Rails.

Reply:
looks correct but I couldn't find the hidden input when you inspect the input when a selection has been made, not sure what we're missing here but we can take a look!

Reply:
Just pushed up a fix.